app/models/twilio_redirection.rb

```diff
@@ -18,6 +18,8 @@ def initialize(location)
       @location = location
     end
 
+    delegate :title, to: :@location
```

Comment:
This is delegated by also you provide a concrete example below that returns an empty string. What's the intent?